docsrc/imap/reference/manpages/systemcommands/cyr_dbtool.rst

```diff
@@ -16,7 +16,7 @@ Synopsis
 
 .. parsed-literal::
 
-    **cyr_dbtool** [ **-C** *config-file* ] [ **-n** ] [ **-o** ] [ **-T** ]
+    **cyr_dbtool** [ **-C** *config-file* ] [ **-n** ] [ **-o** ] [ **-S** ] [ **-N** ] [ **-T** ]
```

Comment:
Option `-R` is not documented.  Can `-o` be removed from here?

Reply:
Yes, have removed -o, documented it, and also fixed EVERYTHING to be documented and sorted case-insignificant alphabetically.